config/rbac/role.yaml

```diff
@@ -4,13 +4,24 @@ kind: ClusterRole
 metadata:
   name: odh-model-controller-role
 rules:
+- apiGroups:
```

Comment:
This file is generated from the  kubebuilder rules in main.go. Any change to the RBAC rules should be applied to main.go. 

https://github.com/opendatahub-io/odh-model-controller/blob/f6fb33a819b8c91c8c08b9612ca8e9ad9282ed84/main.go#L56-L73

Reply:
@vaibhavjainwiz I agreed with @aslakknutsen . Line #73 should be updated.

Reply:
@vaibhavjainwiz do you have bandwidth to do this?